### PR TITLE
Merge alternate branch into master

### PR DESCRIPTION
**Pull request by ssgelm**
This can't be auto merged and probably needs some fixing, but I wanted to start the conversation.  Now that we have found a fuzzy finder that works across vpairs and cpairs, gateway, infra and cosmos repos, I think it's time to merge the alternate branch into master.  What do people think?

cc @pgr0ss @mattikus

### CODE REVIEW

**Comment by pgr0ss** [comment]
I'm in favor of merging the branches together, but it's hard to make sense of this diff to know what is actually changing. Here are a couple of ideas to try to make the diff easier (likely with a few separate PRs):
* Upgrade vim-plug and other dependencies in both branches separately
* Sort `vimrc.bundles` in both branches
* Maybe create a new branch that would be a fast forward from master so people can `git checkout` and try it locally

**Comment by pgr0ss** [comment]
Also, I noticed a few issues in the README while reviewing the diff, so we should make sure https://github.com/braintreeps/vim_dotfiles/pull/64 is merged first or incorporated somehow.

**Comment by ssgelm** [comment]
@pgr0ss Merged.

File: vimrc [hunk 1]
```diff
@@ -132,11 +141,18 @@ let g:netrw_banner = 0
 
 let g:VimuxUseNearestPane = 1
 
-let g:CommandTMaxHeight = 15
-let g:CommandTMatchWindowAtTop = 1
-let g:CommandTCancelMap     = ['<ESC>', '<C-c>']
-let g:CommandTSelectNextMap = ['<C-n>', '<C-j>', '<ESC>OB']
-let g:CommandTSelectPrevMap = ['<C-p>', '<C-k>', '<ESC>OA']
+let $FZF_DEFAULT_COMMAND = 'find * -type f 2>/dev/null | grep -v -E "deps|_build|node_modules|vendor|build_intellij"'
```
Comment (dewyze) [inline vimrc]: @ssgelm can you change this to:

```
let $FZF_DEFAULT_COMMAND = 'find * -type f 2>/dev/null | grep -v -E "deps\/|_build\/|node_modules\/|vendor\/|build_intellij\/"'
```

That way thinks like our `view_builder` don't get ignored. We just want to ignore those folders.

Comment (ssgelm) [inline vimrc]: I didn't make that commit - you'll probably need to update this on the alternate branch so that it gets updated properly.

**Comment by ssgelm** [comment]
Closed in favor of #68

**Pull request closed without merging**